CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added
 
  - Initial version of the `jibri-pulseaudio` docker image based on
-   `jitsi/jibri:stable-5765-1`
+   `jitsi/jibri:stable-6371`
```

Comment:
```suggestion
   `jitsi/jibri:stable-6173`
```

The tags `stable-6371` doesn't exists and you use `6173` in `docker-compose.yml` file

Reply:
oopsie, thanks this is fixed!